data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -49,19 +50,13 @@
 	public List<LineageNodeInfo> getLineageFunctionNodes() {
 		return lineageFunctionNodes;
 	}
-	public LineageQueryInfo getLineageQueryInfo() {
+	/*public LineageQueryInfo getLineageQueryInfo() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)